patches/server/0570-Missing-Entity-API.patch

```diff
@@ -249,6 +249,25 @@ index 37da58385ecf40baeb4665a0d9c262da03c01763..4fa1a12e7e5c23e1d902b59f2a315eac
  
      public Llama(EntityType<? extends Llama> type, Level world) {
          super(type, world);
++        this.maxDomestication = 30; // Paper
+     }
+ 
+     public boolean isTraderLlama() {
+@@ -316,10 +317,14 @@ public class Llama extends AbstractChestedHorse implements VariantHolder<Llama.V
+         return Llama.getDyeColor(this.getItemBySlot(EquipmentSlot.BODY));
+     }
+ 
++    // Paper start - allow api to override
++    /*
+     @Override
+     public int getMaxTemper() {
+         return 30;
```

Comment:
could instead just return super.getMaxTemper() here for
a) less diff
b) documenting that the super method is supposed to be used.